docs/databases/typeorm.md

```diff
@@ -0,0 +1,83 @@
+---
+id: typeorm
+title: TypeORM
+sidebar_label: TypeORM
+---
+
+TypeORM data store for accounts-js
+
+[Github](https://github.com/accounts-js/accounts/tree/master/packages/database-mongo) |
+[npm](https://www.npmjs.com/package/@accounts/typeorm)
+
+## Install
+
+```javascript
+yarn add @accounts/typeorm typeorm pg
+```
+
+## Usage
+
+```typescript
+import AccountsServer from '@accounts/server';
+import typeorm from '@accounts/typeorm';
+import { createConnection } from 'typeorm';
+export const createAccounts = async () => {
```

Comment:
let's add an empty line to make it more readable :)

Reply:
I'm not sure if it was supposed to be '## > Notice' that's what I pushed or just '> Notice'!

but done!

Reply:
Sorry it wasn't clear so I pushed a fix :)